### ISSUE TITLE
gnuradio-3.7.7   GNURADIO_ANALOG_FOUND = FALSE

### ISSUE THREAD

**Issue by tele1** [gnuradio/gr-tutorial#8]
Hi!
I tried compile this source code gr-tutorial
with installed gnuradio-3.7.7 ,
problem is: 

~~~~
...
GNURADIO_ANALOG_FOUND = FALSE
CMake Error at /usr/local/lib64/cmake/gnuradio/GnuradioConfig.cmake:110 (message):
  Required GNU Radio Component: ANALOG missing!
Call Stack (most recent call first):
  /usr/local/lib64/cmake/gnuradio/GnuradioConfig.cmake:118 (GR_MODULE)
  CMakeLists.txt:94 (find_package)
~~~~

Maybe this files exist ? This files ?
~~~~
$ rpm -qpl gnuradio-3.7.7.1-1pclos2015.x86_64.rpm | grep "gr-analog"
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h.html
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h.js
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__dep__incl.map
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__dep__incl.md5
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__dep__incl.png
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__incl.map
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__incl.md5
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h__incl.png
/usr/local/share/doc/gnuradio-3.7.7/html/gr-analog_2include_2gnuradio_2analog_2api_8h_source.html
/usr/local/share/doc/gnuradio-3.7.7/xml/gr-analog_2include_2gnuradio_2analog_2api_8h.xml
~~~~

How check gr-tutorial version ?

**Comment by mbr0wn** [comment]
@tele1 This is not a gr-tutorial bug, it seems like your GNU Radio isn't fully installed. Please ask on the GNU Radio mailing list.

**Comment by tele1** [comment]
I checked in ccmake .. should be compilled .
With older gnu radio I see

~~~~
  Could not find a package configuration file provided by "Gnuradio"
  (requested version 3.7.2)
~~~~

From where I can dowload older gr-tutorial to test ?

**Comment by trondeau** [comment]
@tele1 the instructions say that the Guided Tutorials require version >= 3.7.4.

Please check the output of your cmake (not what you specify, but the output after it runs) to see which components of GNU Radio are being built and make sure gr-analog is one of them.

**Comment by tele1** [comment]
> Please check the output of your cmake (not what you specify, ...
I suggest, something (maybe link, name) was changed in new gnuradio
and cmake from tutorial can not find it, because I'm sure this exist in new version gnuradio,
but I can not fix cmake in  Guided Tutorials, 
I builded rpm, this can be also problem If I used macro, but with macro 
also should work (I don't remember now how I now compiled), 
however gnuradio can find, but not this module. 
However.
Your info about require version is more helpful,
in free time I try compile for friend from forum.
